src/markdown-pages/docs/managers/github-manager.md

```diff
@@ -0,0 +1,290 @@
+---
+path: "/docs/managers/github-manager"
+title: "GitHub Manager"
+---
+
+
+This Manager is at Release level. You can view the <a href="https://javadoc.galasa.dev/dev/galasa/github/package-summary.html">Javadoc documentation for the Manager here</a>.<br>
+
+
+[Overview](#overview)<br>
+[Configuring](#configuring)<br>
+[Provided annotation](#annotations)<br>
+[Code snippets and examples](#codesnippets)<br>
+
+
+# <a name="overview"></a>Overview
+
+
+If there is an open issue in a GitHub repository that tracks a known problem which is expected to cause a test class or test method to fail, you can add the _@GitHubIssue_ annotation to a test class or test method so that the expected failure is recorded as _Failed with Defects_  in the test report. 
+
+Using this annotation on a test class or test method means that the expected failure is clearly identifiable in the test report, making it easier to spot any unexpected test failures and saving time by avoiding the investigation of a known issue. 
+
+
+## Using the Manager
+
+The GitHub Manager is activated if there is a _@GitHubIssue_ annotation on the test class or any of its test methods. The presence of the _@GitHubIssue_ annotation means that there is a known problem which is expected to affect the result of the test. 
+
+If the issue is expected to affect one or more specific test methods, add the annotation on the test method. If the scope of the issue is larger and might affect the whole test class, add the annotation at the class level.
+
+To use the GitHub Manager you must import the _@GitHubIssue_ annotation into the test, as shown in the following example, which is based on the _BatchAccountsOpenTest_ that is provided with Galasa SimBank: 
+
+```
+import dev.galasa.githubissue.GitHubIssue;
+...
+@Test
+@GitHubIssue(issue = "1000", repo = "galasa-dev/projectmanagement")
+public class BatchAccountsOpenTest {
+```
```

Comment:
Since this example is used below do we need it here as well?

Reply:
Will change the code snippets example. I would like to keep an example in both the code snippets and the usage sections. Someone might want to understand how the annotation could be used initially (this section) and then refer straight to the code snippets section to remind themselves how to set it up when they get to that bit.